src/handlers/googleUserContent.ts

```diff
@@ -0,0 +1,29 @@
+export function adjustSize(
```

Comment:
nit but should this be in this file? may be time for a imageUtils

Reply:
those are related to the specific format offered by googleUserContent